docs/content/docs/engines/trino.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Trino"
+weight: 4
+type: docs
+aliases:
+- /engines/trino.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Trino
+
+Because Trino's dependency is JDK 11, it is not possible to include the trino connector in [flink-table-store](https://github.com/apache/flink-table-store).
+
+See [flink-table-store-trino](https://github.com/JingsongLi/flink-table-store-trino).
```

Comment:
It refers to a personal repo, and I'm unsure about the license-related stuff. Is it ok?

Reply:
It is a link instead of dependency. It is OK.